tests/azure/nic_test.go

```diff
@@ -28,7 +28,7 @@ func testNicCreate(t *testing.T, f fixture.F) {
 		f.ResGroupName,
 		f.Location,
 	)
-	_ = azure.NewNsg(f)
+	azure.NewNsg(f)
```

Comment:
Also no need to call any other of the New functions that the return value are ignored, they are basically the same (only the client changes)